api/test/README.md

```diff
@@ -9,7 +9,7 @@ The implementation is <250 loc, (<500 with comments) ( https://github.com/Raynos
 ## Migrating from tape
 
 ```js
-const test = require('tape')
+const tape = require('tape')
 // Tapzero exports an object with a test function property.
 const test = require('tapzero').test
 ```
```

Comment:
not related to this PR, but the readme looks incorrect here. `socket:test` is a built-in module.

Reply:
do we even need this md file?

Reply:
I'm not sure the intention. Maybe we just drop it?

Reply:
Done